build.gradle

```diff
@@ -205,19 +210,14 @@ project(':waltz-server') {
             project(':waltz-storage'),
 
             "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion",
-            "com.wepay.riff:riff-config:$riffVersion",
```

Comment:
Curious about removal of these. Why? I believe these modules are using code from these JARs. Should not depend on transitive dependencies to pull them in.. should be defined explicitly if used in this module.

Reply:
> Should not depend on transitive dependencies to pull them in.. should be defined explicitly if used in this module.
My thought was `waltz-storage` inherits these dependencies from `waltz-common`. But what you say make sense to me, I will add them back!